app/scripts/reducers/map.js

```diff
@@ -1,21 +1,24 @@
-import { COMPLETE_MAP_UPDATE, UPDATE_MAP_DATA, REQUEST_TILE } from '../actions';
+import { COMPLETE_MAP_UPDATE, UPDATE_MAP_DATA, REQUEST_TILE, TOGGLE_EXISTING_ROADS } from '../actions';
 
 const initial = {
   tempStore: null,
-  requestedTiles: new Set()
+  requestedTiles: new Set(),
+  showExistingRoads: false
 };
 
 const map = (state = initial, action) => {
   switch (action.type) {
     case COMPLETE_MAP_UPDATE:
-      return { tempStore: null, requestedTiles: state.requestedTiles };
+      return Object.assign({}, state, { tempStore: null, requestedTiles: state.requestedTiles });
```

Comment:
@sethvincent I was being a little lazy initially (hence the lack of `Object.assign` the first time around). If we're using that syntax now, you can drop the repeated properties (e.g. `requestedTiles` in this first one)